src/test/scala/ensime/consierge/ConfigFetcherTest.scala

```diff
@@ -0,0 +1,18 @@
+package ensime.consierge
+
+import org.scalatest.{FlatSpec, Matchers}
+import scala.concurrent.duration._
+
+class ConfigFetcherTest extends FlatSpec with Matchers {
+
+  "config" should "load all data from test file" in {
+    val config = Configuration.load
+    assert(config.owner === "user_name")
```

Comment:
Have you seen the `should equal` style? It's really nice, I recommend reading the full ScalaTest guide, it is incredibly well designed.